src/finufft.cpp

```diff
@@ -645,17 +649,16 @@ int FINUFFT_MAKEPLAN(int type, int dim, BIGINT* n_modes, int iflag,
     int nthr_fft = nthr;    // give FFTW all threads (or use o.spread_thread?)
                             // Note: batchSize not used since might be only 1.
     // Now place FFTW initialization in a lock, courtesy of OMP. Makes FINUFFT
-    // thread-safe (can be called inside OMP) if -DFFTW_PLAN_SAFE used...
-#pragma omp critical
+    // thread-safe (can be called inside OMP)
     {
```

Comment:
This "did_fftw_init" was my own global state for finufft to make sure fftw init only once. Is it correct, now you've thought about fftw deeply?